src/UCommerce.SiteCore.Installer/Steps/MoveResourceFiles.cs

```diff
@@ -20,7 +20,8 @@ public MoveResourceFiles(DirectoryInfo basePath, IInstallerLoggingService loggin
 
         public Task Run()
         {
-            var postInstallationSteps = new List<IStep>();
+            _loggingService.Information<MoveResourceFiles>("Starting movement of resource files.");

```

Comment:
Could this perhaps be better like this
```suggestion
            _loggingService.Information<MoveResourceFiles>($"Moving resource files from {SourceDirectory.FullName}...");

```